packages/builder/src/constants/backend/index.js

```diff
@@ -165,20 +165,11 @@ export const FIELDS = {
       BBReferenceFieldSubType.USER
     ],
   },
-  // Used for display of editing existing columns
-  DEPRECATED_USER: {
-    name: "User",
-    type: FieldType.BB_REFERENCE,
-    subtype: BBReferenceFieldSubType.USER,
-    icon: TypeIconMap[FieldType.BB_REFERENCE_SINGLE][
-      BBReferenceFieldSubType.USER
-    ],
-  },
   USERS: {
     name: "User List",
     type: FieldType.BB_REFERENCE,
-    subtype: BBReferenceFieldSubType.USERS,
-    icon: TypeIconMap[FieldType.BB_REFERENCE][BBReferenceFieldSubType.USERS],
```

Comment:
I'm not sure I understand these changes. Don't we need to keep this old field?

Reply:
This was added to be able to display the old columns properly. This might be confusing as it's not clearly used anywhere (because it's consumed dynamically). Instead, I handled these cases in the actually usage (in `packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte`